requirements.txt

```diff
@@ -11,4 +11,5 @@ Flask>=1.1.2
 mtcnn>=0.1.0
 lightgbm>=2.3.1
 dlib>=19.20.0
-retina-face>=0.0.1
\ No newline at end of file
+retina-face>=0.0.1
+matplotlib==3.4.3
```

Comment:
I've added the latest matplotlib as it can come in handy for debugging. It might later be moved to 'test only' dependencies.